Marble Blast Platinum/platinum/client/scripts/audioprofiles.cs

```diff
@@ -328,7 +328,7 @@
 }
 
 function playGameMusic() {
-	if (MissionInfo.music !$= "" && MissionInfo.music !$= "Pianoforte.ogg") {
+	if (MissionInfo.music !$= "") {
```

Comment:
Seems reasonable, but is there a historical reason why it was preventing maps from playing this specific track?

Reply:
I assume it was because Pianoforte was the menu music, and as such shouldn't be allowed to play in a level. However, it seems like an arbitrary restriction nowadays, especially considering that the menu music gets replaced after beating PlatinumQuest (and, unlike Pianoforte, Electroforte is not restricted in the same way).

Reply:
That makes sense to me. I just want to make sure that there is no legacy behavior that's going to be broken because of this, because you never know

Reply:
I don't _think_ that there will be any problems, this is the only place where it checks and for Pianoforte and does something based on that (unless I'm missing some super secret function)

Reply:
![image](https://user-images.githubusercontent.com/7307599/231652241-7b5a78ef-34b2-43b2-930f-0befeee07160.png)

Hmm, it seems there are a substantial number of missions which have the music set to pianoforte. Maybe it was the default at some point? Regardless, I'm pretty sure that line of code is there to make sure the music have more variety in those levels. For now, we probably shouldn't include this change. Even if we went through and changed all of the missions files that show up in the search, there might be unexpected changes on marbleland levels.

It'd be nice to have some way of selecting pianoforte as the music track for a level, but I don't think removing that check is the best way. Even if I'd say it's the "proper" fix, the levels that we got are gonna make it a headache.

Reply:
*bang*
Well if it isn't allowed in levels, it shouldn't appear as a valid song inside editmissioninfo.